_pages/documentation/general_docs/architecture_support/arm_implementation.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: documentation
+title: "ARM implementation"
+doc: ARM implementation
+parent: architecture_support
+permalink: /documentation/general_docs/architecture_support/arm_implementation
+author: "?"
```

Comment:
Could you drop this line? I think it will render as "author: ?" on the page, and I would rather just have no author.